types/node/v10/node-tests.ts

```diff
@@ -952,6 +952,10 @@ function bufferTests() {
         const teEncodeRes: Uint8Array = te.encode("TextEncoder");
 
         // util.types
+        let b: Boolean;
+        b = util.types.isBigInt64Array(15);
+        b = util.types.isBigUint64Array(15);
```

Comment:
Can you add a type guard test here, I'm worried TS won't do the type assertion correctly here.

Reply:
done in ts 3.2. specific test as the generic one doesn't know about `BigInt64Array` and therefore can't do a type guard.
During playing around I found that the type guard for `isBoxedPrimitive()` is actually not really asserting anything useful but I failed to repair it. Seems typescript can't type guard correct with unions of boxed types.